internal/exec/stages/files/passwd.go

```diff
@@ -77,6 +78,23 @@ func (s *stage) createPasswd(config types.Config) error {
 	return nil
 }
 
+func userUIDConflict(a types.PasswdUser, list []types.PasswdUser) error {
```

Comment:
I might be missing something, but I don't think this function is necessary, since `useradd` will catch duplicates for us.  And `userUIDConflict` won't catch conflicts with users that already exist on the system.